src/routes/authors.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const { 
+   createAuthor, 
+   readAuthors, 
+   getAuthorById, 
+   updateAuthor, 
+   deleteAuthor 
+} = require('../controllers/authors');
+
+const authorRouter = express.Router();
+
+authorRouter.route('/')
+.post(createAuthor)
+.get(readAuthors);
```

Comment:
For shorter chained commands like this you can put them all on one line like this:
```
authorRouter.route('/').post(createAuthor).get(readAuthors);
```